docs/topics/auth/default.txt

```diff
@@ -1611,20 +1611,20 @@ the logged-in user has any permissions in the ``foo`` app::
 
 Evaluating a two-level-attribute lookup as a boolean is a proxy to
 :meth:`User.has_perm() <django.contrib.auth.models.User.has_perm>`. For example,
-to check if the logged-in user has the permission ``foo.can_vote``::
+to check if the logged-in user has the permission ``foo.add_vote``::
```

Comment:
Why have you changed these `foo` permissions? 

(I don't have an opinion really... just asking)

Reply:
I think it was on the last pr.
Maybe to be compatible with others.